Description: Change version of dependencies to one available in Debian
Author: Komal Sukhani <komaldsukhani@gmail.com> 
Forwarded: no 
Last-Update: 2015-06-22
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: debian-groovy2/subprojects/groovy-bsf/build.gradle
===================================================================
--- debian-groovy2.orig/subprojects/groovy-bsf/build.gradle
+++ debian-groovy2/subprojects/groovy-bsf/build.gradle
@@ -2,7 +2,7 @@ dependencies {
     compile('bsf:bsf:2.4.0') {
         exclude(group: 'commons-logging', module: 'commons-logging')
     }
-    compile 'commons-logging:commons-logging:1.2'
+    compile 'commons-logging:commons-logging:debian'
     compile rootProject
     testCompile project(':groovy-test')
 }
